itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -1256,6 +1246,7 @@ public void testEventTypesForDynamicAddPartitionByInsert() throws IOException {
 
     String[] ptn_data = new String[]{ "ten"};
     run("INSERT INTO TABLE " + dbName + ".ptned partition(b=1) values('" + ptn_data[0] + "')");
+    run("DROP TABLE " + dbName + ".ptned");
```

Comment:
It doesn't modify the existing test which checks that no INSERT event is generated by dynamic inserts. I added DROP TABLE explicitly to avoid a failure caused by another known bug (ALTER_PARTITION event doesn't generate any task during REPL LOAD if partition is missing due to which the last repl ID returned by REPL DUMP doesn't match with the one returned by REPL STATUS). Dynamic inserts generate ADD_PARTITION and ALTER_PARTITION events. Just to avoid ALTER_PARTITION as last event in the dump, I added DROP.